app/src/main/kotlin/com/mapzen/erasermap/controller/MainActivity.kt

```diff
@@ -587,6 +585,10 @@ public class MainActivity : AppCompatActivity(), MainViewController, RouteCallba
     }
 
     override fun showReverseGeocodeFeature(features: List<Feature>) {
+        hideSearchResults()
+        centerOnCurrentFeature(features)
```

Comment:
I don't think we need to but it definitely feels nicer as a user when it centers. @mjcunningham want to weigh in?